server/jobs/job_receiver_registry.go

```diff
@@ -0,0 +1,58 @@
+package jobs
+
+import "sync"
+
+type ReceiverRegistry interface {
```

Comment:
can probably just name this file `receiver_registry.go`

Reply:
Done!